### ISSUE TITLE
Uncaught TypeError: Cannot read property 'classList' of undefined

### ISSUE THREAD

**Issue by talshorer** [amitab/atom-cscope#30]
[Enter steps to reproduce:]

1. Look for a symbol that only produces one entry (e.g. global symbol search for a function)

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: Ubuntu 16.04.2
**Thrown From**: [atom-cscope](https://github.com/amitab/atom-cscope) package 1.0.0

### Stack Trace

Uncaught TypeError: Cannot read property 'classList' of undefined

```
At /home/tal/.atom/packages/atom-cscope/lib/views/atom-cscope-view.coffee:76

TypeError: Cannot read property 'classList' of undefined
    at AtomCscopeView.module.exports.AtomCscopeView.selectNext (/packages/atom-cscope/lib/views/atom-cscope-view.coffee:76:49)
    at /packages/atom-cscope/lib/viewModels/atom-cscope-view-model.coffee:65:13)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:265:35)
    at /usr/share/atom/resources/app/src/command-registry.js:3:65
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:610:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:401:28)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/usr/share/atom/resources/app/src/window-event-handler.js:100:42)
    at HTMLDocument.<anonymous> (/usr/share/atom/resources/app/src/window-event-handler.js:3:65)
```

### Commands

```
     -0:40.9.0 editor:move-to-previous-subword-boundary (input.hidden-input)
  3x -0:40.5.0 editor:select-to-next-subword-boundary (input.hidden-input)
     -0:39.5.0 core:copy (input.hidden-input)
     -0:39.2.0 atom-cscope:toggle (input.hidden-input)
     -0:38.1.0 core:select-all (input.hidden-input)
     -0:38 core:paste (input.hidden-input)
     -0:37.1.0 core:confirm (input.hidden-input)
  5x -0:35.9.0 core:move-down (input.hidden-input)
  4x -0:09.9.0 core:move-up (input.hidden-input)
  3x -0:08.9.0 core:move-down (input.hidden-input)
     -0:05.1.0 core:confirm (input.hidden-input)
     -0:04.6.0 core:move-down (input.hidden-input)
     -0:03.8.0 editor:consolidate-selections (input.hidden-input)
     -0:03.8.0 core:cancel (input.hidden-input)
     -0:02.7.0 atom-cscope:toggle (input.hidden-input)
     -0:02.1.0 core:move-down (input.hidden-input)
```

### Non-Core Packages

```
atom-cscope 1.0.0 
highlight-selected 0.13.1 
jumpy 3.1.3 
multi-cursor-plus 1.2.0 
open-files 0.8.2 
tab-smart-sort 0.3.0 
```

```

**Comment by mfiore02** [comment]
Bump. I'm also experiencing this issue. After searching for a symbol that produces one result, I try to <down arrow> <enter> to jump to that result and get the error listed above.

**Comment by amitab** [comment]
Alright, definitely a bug. I'm working on this fix.

**Comment by amitab** [comment]
Fixed in commit bed29e2d639037aab41e76b441260de5c312aa85
